.github/workflows/ci.yaml

```diff
@@ -17,8 +17,10 @@ jobs:
       - name: Checkout
         uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v3.0.2
 
+      # If we want to have image locally instead of build+push, we can't use multi-platform (docker limitation)
+      # If we want multi-platform smoke, we'd have to either push to registry, or build 2 separate images
```

Comment:
Let's build 2 separate images please, same as we do in Ingress

Reply:
Done